.gitignore

```diff
@@ -9,3 +9,9 @@ target
 .DS_Store
 *.bpmn/
 /userguide/src/en/index.html
+
+\.metadata/
+
+\.recommenders/
+
+modules/activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter-basic/\.factorypath

```

Comment:
It's a very trivial problem, but is it correct to specify only the files of this project?

Reply:
I am using GitKraken to deal with Git. By default it puts directives to ignore specific files. I fixed it.